src/adminjs.spec.ts

```diff
@@ -49,6 +49,41 @@ describe('AdminJS', function () {
     })
   })
 
+  describe('resolveBabelConfigPath', function () {
+    it('load .babelrc file', function () {
+      const adminBro = new AdminJS({ bundler: { babelConfig: '../.babelrc' } })
```

Comment:
Please change the const name to `adminJs` for consistency (do that for other tests too)